google/cloud/dialogflow_cx/session_entity_types_client.h

```diff
@@ -96,17 +96,7 @@ class SessionEntityTypesClient {
   /// Returns the list of all session entity types in the specified session.
   ///
   /// @param parent  Required. The session to list all session entity types from.
-  ///  Format:
-  ///  @n
-  ///  @code
-  ///  projects/<Project ID>/locations/<Location ID>/agents/<Agent ID>/sessions/<Session ID>
-  ///  @endcode
-  ///  @n
-  ///  or
-  ///  @n
-  ///  @code
-  ///  projects/<Project ID>/locations/<Location ID>/agents/<Agent ID>/environments/<Environment ID>/sessions/<Session ID>
-  ///  @endcode
+  ///  Format: `projects/<Project ID>/locations/<Location ID>/agents/<Agent ID>/sessions/<Session ID>` or `projects/<Project ID>/locations/<Location ID>/agents/<Agent ID>/environments/<Environment ID>/sessions/<Session ID>`.
```

Comment:
I wonder how these render in the final docfx format.  Nothing to do but wait and fine tune if needed.

Reply:
Yeah, the final rendering may be the same, or it may be slightly worse.

Still, the service will probably add new RPCs with similar comments. I think we will spend less time in the future fixing them.